float32 angle_min
float32 angle_increment
float32[] x
float32[] y
int32[] label